proposed/http-client/http-client.md

```diff
@@ -78,7 +78,7 @@ interface ClientInterface
      *
      * @throws \Psr\Http\Client\Exception If an error happens during processing the request.
      */
-    public function sendRequest(RequestInterface $request);
+    public function sendRequest(RequestInterface $request): Response;
```

Comment:
Shouldn't this be `ResponseInterface` as per docblock?

Reply:
You are very much correct. Thank you!